SimPEG/Examples/EM_NSEM_3D_Foward.py

```diff
@@ -69,12 +75,43 @@ def run(plotIt=True, nFreq=1):
     fields = problem.fields()
     dataVec = survey.eval(fields)
 
-    # Make the data
-    mtData = NSEM.Data(survey, dataVec)
+    # Add uncertainty to the data - 10% standard
+    # devation and 0 floor
+    dataVec.standard_deviation.fromvec(
+        np.ones_like(simpeg.mkvc(dataVec)) * 0.1
+    )
+    dataVec.floor.fromvec(
+        np.zeros_like(simpeg.mkvc(dataVec))
+    )
 
     # Add plots
     if plotIt:
-        pass
+        # Plot the data
+        # On and off diagonal (on left and right axis, respectively)
+        fig, axes = plt.subplots(2, 1, figsize=(7, 5))
+        plt.subplots_adjust(right=0.8)
+        [(ax.invert_xaxis(), ax.set_xscale('log')) for ax in axes]
+        ax_r, ax_p = axes
+        ax_r.set_yscale('log')
+        ax_r.set_ylabel('Apparent resistivity [xy-yx]')
+        ax_r_on = ax_r.twinx()
+        ax_r_on.set_yscale('log')
+        ax_r_on.set_ylabel('Apparent resistivity [xx-yy]')
+        ax_p.set_ylabel('Apparent phase')
+        ax_p.set_xlabel('Frequency [Hz]')
+        # Start plotting
+        ax_r = dataVec.plot_app_res(
+            np.array([-200, 0]),
+            components=['xy', 'yx'], ax=ax_r, errorbars=True)
+        ax_r_on = dataVec.plot_app_res(
+            np.array([-200, 0]),
+            components=['xx', 'yy'], ax=ax_r_on, errorbars=True)
+        ax_p = dataVec.plot_app_phs(
+            np.array([-200, 0]),
+            components=['xx', 'xy', 'yx', 'yy'], ax=ax_p, errorbars=True)
+        ax_p.legend(bbox_to_anchor=(1.05, 1), loc=2)
+
+        plt.show()
```

Comment:
Right now the testing is running this, can you put the `plt.show()` beneath `if name == '__main__'` - that should do the trick!